Resources/Private/Language/de.locallang_em.xlf

```diff
@@ -21,6 +21,18 @@
             <trans-unit id="authorization.accessTokenSecret">
                 <target>Access token secret</target>
             </trans-unit>
+            <trans-unit id="authorization.baseUrl">
```

Comment:
There is no `locallang_em.xlf` file. Maybe you should rename the file to `de.locallang_mod.xlf`?